Comments.md

```diff
@@ -124,3 +124,61 @@ func loadLastCheckpoint() -> GameState? {
     return loadedState
 }
 ```
+
+## Convention: Single line and multi line comment types
```

Comment:
Something I started reviewing and apparently didn't publish: Should we explicitly mention that for non-public properties we recommend using /// as well? Or do we have no recommendation? In which case that would also be good to explicitly state?

This documentation is awesome, but the main remaining question that usually comes up is "what should I do for comments in code which don't directly translate to public API documentation?